File: C_Programming/Competitive_Programming/CP_010.c
////////////////////////////////////////////////////////////////////
//
//  File Name    : Check_Even_Or_Odd
//
//  Description  : This program accepts a number from the user
//                 and checks whether the number is even or odd.
//
//  Example:
//      Input  : 6
//      Output : Given number is an Even Number
//
//      Input  : 11
//      Output : Given number is an Odd Number
//
////////////////////////////////////////////////////////////////////

#include<stdio.h>

#define TRUE 1
#define FALSE 0

typedef int BOOL;

BOOL ChkEven(int iNo)
{
    if(iNo % 2 == 0)
    {
        return TRUE;
    }
    else
    {
        return FALSE;
    }    
}

int main()
{
    int iValue = 0;
    BOOL bRet = FALSE;

    printf("Enter Number : \n");
    scanf("%d", &iValue);

    bRet = ChkEven(iValue);

    if(bRet == TRUE)
    {
        printf("Given number is a Even Number");
    }
    else
    {
        printf("Given number is not a Even Number");
    }  

    return 0;
}

File: Java_Programming/Competitive_Programming/CP_010.java
//////////////////////////////////////////////////////////////////
//
//  File Name    : Check_Number_Sign
//
//  Description  : This program accepts a number and
//                 checks whether the given number is
//                 positive, negative, or zero.
//
//  Example:
//      Input  : -8
//      Output : Number is Negative Number.
//
//////////////////////////////////////////////////////////////////

class Logic
{
    void checkSign(int iNum)
    {
        if(iNum > 0)
        {
            System.out.println("Number is Positive Number.");
        }
        else if(iNum < 0)
        {
            System.out.println("Number is Negative Number.");
        }
        else
        {
            System.out.println("Number is Zero.");
        }    
    }
}

class CP_010
{
    public static void main(String A[])
    {
        Logic lobj = new Logic();

        lobj.checkSign(-8);
    }
}